syntax="proto3";
package fraft;

message ServerInfo
{
  string server_host = 1;
  uint32 leader_port = 2;
  uint32 election_port = 3;
  uint64 server_id = 4;
};

message ServerConfig
{
  uint64 my_server_id = 1;
  repeated ServerInfo server_info = 2;
};
